openmdao/utils/reports_system.py

```diff
@@ -46,7 +46,7 @@ def reports_active():
     bool
         Return True if reports are active.
     """
-    return os.environ.get('TESTFLO_RUNNING', '').lower() not in ('1', 'true')
+    return os.environ.get('TESTFLO_RUNNING', '').lower() not in _truthy
```

Comment:
I wonder if we need `_truthy` and `_falsey` to exist here in `reports_system`... it seems redundant with `general_utils.env_truthy`.
This version of `_truthy` is now being imported and used in `Problem` as well... seems like we should just be using the existing logic from `general_utils`...

Reply:
I changed over to using general_utils functions.